pkg/controller/launcher/job_test.go

```diff
@@ -13,15 +14,14 @@ import (
 )
 
 const numInitContainers = 4
-const imageTag = "sha256:123456789"
-const imageURI = "quay.io/my-repo/cluster-curator-controller"
+const imageURI = "quay.io/my-repo/cluster-curator-controller@sha123456789"
```

Comment:
Just a question for my own edification: why is it OK to use a hard coded imageURI like this in a testing context. To me, perhaps naively, it seems more reasonable to take it in as an environment variable, as down the line the cluster-curator-controller version you want to test here may change...I'm probably missing something though. Thanks.

Reply:
@Randy424  good question. There is no right answers, but I think it's pretty common to use some hard coded string for unit tests. 

Hard coded string in unit tests is efficient, easy to change, and easy to read. We can create different type of hard-coded test cases for different input very quickly (e.g. empty string/invalid string/...) 

Using an env var can be overkills for unit tests, as it requires user env settings. Here the function is just expecting a string instead of reading an env var, so should be good. 👍